SU2_CFD/src/numerics_direct_turbulent.cpp

```diff
@@ -534,6 +534,14 @@ void CSourcePieceWise_TurbSA::ComputeResidual(su2double *val_residual, su2double
 //  AD::SetPreaccIn(TurbVar_i[0]);
 //  AD::SetPreaccIn(TurbVar_Grad_i[0], nDim);
 //  AD::SetPreaccIn(Volume); AD::SetPreaccIn(dist_i);
+  
+//Added by Cakmakcioglu, for BC Transition Model 10/30/2106
+	bool transition    = (config->GetKind_Trans_Model() == BC);
```

Comment:
Maybe add this variable as a field to the numerics class and set it in the constructor (then it is consistent).